src/main/java/cyrus/adapters/LocalDateAdapter.java

```diff
@@ -0,0 +1,42 @@
+package cyrus.adapters;
+
+import java.io.IOException;
+import java.time.LocalDate;
+
+import com.google.gson.TypeAdapter;
+import com.google.gson.stream.JsonReader;
+import com.google.gson.stream.JsonWriter;
+
+import cyrus.utility.DateUtility;
+
+/**
+ * GSON adapter to properly handle {@code LocalDate} values in the various {@code Task}.
+ */
+public class LocalDateAdapter extends TypeAdapter<LocalDate> {
+    /**
+     * When writing {@code LocalDateAdapter} to JSON, convert it to the input format of {@code dd
```

Comment:
Perhaps starting with a continuous verb could better adhere to JavaDoc conventions? E.g. "Converts the code from..."